### PR TITLE
feat(HMSPROV-428): Add provisioning dashboard

### PR DESCRIPTION
**Pull request by adiabramovitch**
You can see the dashboard locally by importing it into Grafana. 
After this PR gets merged,  MR to app-stre needs to be pushed and merged to enable the dashboard deployment. 
_Reminder: It is the first version of the dashboard_

### CODE REVIEW

File: deploy/dashboards/grafana-dashboard-provisioning.configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,1353 @@
...
+            },
+            "hide": 0,
+            "includeAll": false,
+            "multi": false,
+            "name": "datasource",
+            "options": [],
+            "query": "prometheus",
+            "queryValue": "",
+            "refresh": 1,
+            "regex": "",
```
[the remark this answers is not in the document]
Comment (adiabramovitch) [reply deploy/dashboards/grafana-dashboard-provisioning.configmap.yaml:1254]: Yes, you were right, it was missing here: 
image
Thanks for noticing

**Comment by ezr-ondrej** [comment]
There is no code change, so I'll merge this even with failing tests, the pipeline is broken, no reason to block no-code changes :)

**Review by ezr-ondrej — APPROVED** [review]
Thanks @adiabramovitch! Let's deploy this and see it in action :tada: 
:shipit:

**Pull request merged**